calculator/main.py

```diff
@@ -1,19 +1,20 @@
 from datetime import datetime
 
-from calculator.operations import *
-from calculator.exceptions import *
+from .operations import *
+from .exceptions import *
 
 
-def create_new_calculator(operations=None):
+def create_new_calculator(operations={}):
```

Comment:
This is a bad idea. Subtle (and advanced) bug, but here is for you guys to read about it (Mistake 1 in this list): https://www.toptal.com/python/top-10-mistakes-that-python-programmers-make